### PR TITLE
Remove unused dependence on GEANT3 common blocks

### PR DESCRIPTION
**Pull request by klendathu2k**
Remove unused dependence on the GEANT3 FORtran common blocks (mapped to c-structs), and remove the definitions contained in G3Commons.h.  These definitions are redundant, and can cause conflicts in ROOT6 / cling, with the common blocks defined in the TGeant3.h VMC header.

### CODE REVIEW

**Review by akioogawa — APPROVED** [review]

File: StarVMC/StarAgmlLib/AgModule.cxx [hunk 1]
```diff
@@ -45,57 +45,14 @@ Bool_t AgModule::AddHit( string _for, string meas, Float_t bitns, Float_t mn, Fl
...
-    {
-      //      cout << "Register new cut table for block " << block.c_str() << endl;
-      // Create the table and add default set of cuts
-      THEBLOCK -> Add( G3cuts = new TGenericTable( "Gccuts_t", "G3cuts" ) );
-      Gccuts_t cuts; /* new default cuts */ 
-      G3cuts -> AddAt( &cuts );
-    }
-
-  // Set new cut value for this cut
-  ((Gccuts_t *)G3cuts->At(0))->set( cut, value );
```
Comment (veprbl) [inline StarVMC/StarAgmlLib/AgModule.cxx:64]: Why not nuke the function altogether, or, at least, make it call `std::abort()`?

Comment (klendathu2k) [reply StarVMC/StarAgmlLib/AgModule.cxx:64]: Ok

**Review by plexoos — DISMISSED** [review]

**Comment by klendathu2k** [comment]
Looks like this requires modification of the agml parser code.  Will attend to once I'm back.

**Comment by klendathu2k** [comment]
@plexoos FYI -- fixed the issue with the agmlParser, and the code is now passing checks.

**Review by plexoos — APPROVED** [review]
LGTM

**Comment by genevb** [comment]
This PR failed the DEV AutoBuild (Issue #258 ).

**Pull request merged**